.pa11yci--mobile

```diff
@@ -2,9 +2,9 @@
   "defaults": {
     "userAgent": "pa11y",
     "timeout": 120000,
-    "concurrency": 2,
```

Comment:
Higher concurrency resulted in more `Failed to run` errors. With pa11y 3, the default is `"concurrency":1`. Reference:[ Pa11y-ci 3 migration guide](https://github.com/pa11y/pa11y-ci/blob/master/MIGRATION.md)